CMakeLists.txt

```diff
@@ -107,7 +107,9 @@ mark_as_advanced(TEST_SUITE_REMOTE_CLIENT)
 if(TEST_SUITE_REMOTE_HOST)
   add_custom_target(rsync
     COMMAND ${PROJECT_SOURCE_DIR}/utils/rsync.sh
-                ${TEST_SUITE_REMOTE_HOST} ${PROJECT_BINARY_DIR}
+                ${TEST_SUITE_REMOTE_CLIENT}
```

Comment:
Do we need to add a default for this somewhere?

Reply:
It is an already existing variable. It is declared at https://github.com/llvm/llvm-test-suite/blob/main/CMakeLists.txt#L103 and defaults to `ssh`.

Reply:
Sounds good, thanks!